storage/job.go

```diff
@@ -133,15 +134,15 @@ type AddJobInputsInput struct {
 
 // AddJobInputs submits inputs to an already created job.
 func (s *JobClient) AddInputs(ctx context.Context, input *AddJobInputsInput) error {
-	path := fmt.Sprintf("/%s/jobs/%s/live/in", s.client.AccountName, input.JobID)
+	fullPath := path.Join("/", s.client.AccountName, "jobs", input.JobID, "live/in")
```

Comment:
Change this from `"live/in"` to `"live", "in"` so that if we ever add escaping and change `path.Join()` with something else, we don't have to hunt down why `live/in` was double-escaped to `"live%20in"`.